frontend-styleguide/README.md

```diff
@@ -488,3 +489,69 @@ it("Correctly formats the name", () => {
 ```
 
 [Back to top ⬆️](#pushpin-summary)
+
+
+## Translate
+
+### 👉 Use I18n to translate
+
+Our plataform has support for internationalized (i18n). First step is configurated [nextJs](https://nextjs.org/docs/advanced-features/i18n-routing) to support locales on the routes.
+
+Second step, is configure lib [next-translate](https://github.com/vinissimus/next-translate), the plugin is used to render keys into the pages on application.
+
+file `next.config.js`
+```js
+const nextTranslate = require('next-translate');
+
+module.exports = nextTranslate({
+  i18n: {
+    locales: ['pt-BR', 'en'],
+    defaultLocale: 'pt-BR',
+  },
+});
+```
```

Comment:
I wouldn't break this into first & second steps, since they area already configured into the applications and we don't need to do this regularly or ever again (unless we create another repo).

Maybe it's better to just say that we use nextjs built-in i18n and next-translate to support rendering tokens on pages.